packages/security/test/SecurityService.ts

```diff
@@ -0,0 +1,35 @@
+import { assert, expect } from "chai";
+import mocha from "mocha";
+
+import { SecurityService } from "../src";
+import { MockCheck } from "./MockCheck";
+
+let securityService: SecurityService;
+
+describe("SecurityService", () => {
+
+  beforeEach(() => {
+    securityService = new SecurityService();
+  });
+
+  describe("#test", () => {
+
+    it("should pass", (done) => {
```

Comment:
I would extend the tests name more. Each `it` should describe a test case, not the result. _It should pass if the check passes_ for example.